Sources/ImageKit/Util/UploadApi.swift

```diff
@@ -8,12 +8,20 @@
 import Foundation
 import OSLog
 
-class UploadAPI: NSObject, URLSessionTaskDelegate {
-    internal static var baseUrl = "https://upload.imagekit.io"
+fileprivate class SendableAny : @unchecked Sendable {
+    let value: Any
+    
+    init(value: Any) {
+        self.value = value
+    }
+}
```

Comment:
By doing [this](https://github.com/imagekit-developer/imagekit-ios/pull/29/files#r2454809821) can we eliminate the use of SendableAny?

Reply:
@ahnv while we can eliminate the use for upload extensions by this method, we cannot do the same for the _customMetadata_ dictionary, which can have the values of varying types. So we still need the _SendableAny_ type to handle that case.